api/server/router/local/image.go

```diff
@@ -292,9 +294,14 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 		// Do not write the error in the http output if it's still empty.
 		// This prevents from writing a 200(OK) when there is an interal error.
 		if !output.Flushed() {
-			return err
+			return fmt.Errorf("%s\n%s", err.Error(), notVerboseBuffer)
```

Comment:
Since you're calling `err.Error()` here and not just returning `err`, can you move the call to GetErrorString() on line 300 up to before this `return` and then use `errString` instead?